modules/calib3d/test/test_cameracalibration_badarg.cpp

```diff
@@ -489,7 +489,14 @@ class CV_ProjectPoints2BadArgTest : public cvtest::BadArgTest
     void run(int /* start_from */ )
     {
         CvMat zeros;
+#if defined __GNUC__ && __GNUC__ >= 8
```

Comment:
I don't want to introduce macros for this.
Compiler is correct: we can't use `memset()` here (due using of non POD structures).
This code must be get fixed. So I prefer to avoid introduction of these macros - we should not use these macro at all.
Unfortunately, proper fix for these lines is huge: #11630